site/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "AGPL-3.0",
   "scripts": {
     "postinstall": "pnpm typegen",
-    "build": "NODE_ENV=production pnpm vite build",
+    "build": "NODE_ENV=production NODE_OPTIONS=--max-old-space-size=8192 pnpm vite build",
```

Comment:
Wondering if this is something we should set here or if this should be set by the user in their environment which I think would be better. Let me get another opinion on this.